pyfar/dsp/dsp.py

```diff
@@ -1487,4 +1495,194 @@ def decibel(signal, domain='freq', log_prefix=None, log_reference=1):
+
+    ``signal_normalized = signal * target / reference``,
+
+    where `target` equals
+    1 and `reference` is the maximum absolute amplitude before the
+    normalization.
+
+    It is for example also possible normalize to the mean absolute amplitude or
+    to normalize the magnitude spectrum within a certain frequency range. This
+    is done as described above, but changes the way in which `reference` is
+    calculated.
+
+    Parameters
+    ----------
+    signal: Signal
+        Input signal of the signal class.
+    domain: string
+        The domain in which the signal is normalized
+
+        ``'time'``
+            Normalize the absolute time data.
+        ``'freq'``
+           Normalize the absolute frequency data. In this case the magnitude
+           spectrum with FFT normalization is used (``signal.freq``, cf.
+           :py:mod:`FFT concepts <pyfar._concepts.fft>`).
+
+        The default is ``'time'``.
+    dB: bool, optional
+        If `dB` is ``True``, the logarithmic magnitude spectra are used for
+        normalization. They are calculated by :py:func:`~pyfar.dsp.decibel`.
+        The default is ``False``.
+    power: bool, optional
+        If `power` is ``True``, the power spectra :math:`|X|^2` will be used
+        for normalization. Notice that `power` does have no effect if `dB` is
+        ``True``.
+    operation: string, optional
+        Operation to compute the `reference` value described above
+
+        ``'max'``
+            Takes the absolute maximum value of each channel.
+        ``'mean'``
+            Takes the absolute mean value of each channel.
+        ``'rms'``
+            Takes the root mean square value of each channel.
+
+        The default is ``'max'``
+    channel_handling: string, optional
+        Define how multi-channel signals are handled. This parameter does
+        not affect single-channel signals.
+
+        ``'each'``
+            Separate normalization of each channel.
+        ``'max'``
+            Normalize to the maximum `reference` value across channels.
+        ``'min'``
+            Normalize to the minimum `reference` value across channels.
+        ``'mean'``
+            Normalize to the mean `reference` value across the channels.
+
+       The default is ``'max'``
+    target: scalar, array
+        The target to which the signal is normalized in the unit according to
+        `dB`. Can be a scalar or an array. In the latter case the shape of
+        `target` must be broadcasted ``signal.cshape``. The default is 0 dB if
+        ``dB=True`` and 1 otherwise.
+    freq_range: tuple
+        Two element vector specifying upper and lower frequency in Hz between
+        which the `reference` value for normalization is computed.
+
+    Returns
+    --------
+    normalized_signal: Signal
+        The normalized signal
+    reference: numpy array
+        If ``return_reference=True`` the function also return the `reference`
+        values described above.
+    """
```

Comment:
I think it would be cool to just list some typical normalizations as Examples here :)

Reply:
Since the dB parameter was removed, an example incl. ``decibel`` would be nice, too.